custom/icds_reports/sqldata/base_populations.py

```diff
@@ -25,3 +28,29 @@ def rows(self):
                     data['open_count']
                 ]
             ]
+
+
+class BasePopulationBeta(ICDSMixin):
+
+    slug = 'population'
+
+    @property
+    def headers(self):
+        return []
+
+    @property
+    def rows(self):
+        if self.config['location_id']:
+            filters = get_location_filter(self.config['location_id'], self.config['domain'])
+
+            if filters.get('aggregation_level')>1:
+                filters['aggregation_level'] -= 1
+
+            filters['month'] = date(self.config['year'], self.config['month'], 1)
+            awc_data = AggAwc.objects.filter(**filters).values('cases_person').order_by('month').first()
```

Comment:
i think this changes the meaning of that indicator slightly (includes a lot more conditions)

Reply:
Hey yes, There are some minute changes done on the definition of the indicator both in order to make sense out of them also to align with the dashboard.
you can read about these in the doc attached here: https://app.asana.com/0/1188875673564905/1170991451023294